src/FlowRules/Types/type_utils.jl

```diff
@@ -38,10 +38,11 @@ function setup_well_model(M::jutulModel{D, T}, f::Union{jutulForce{D, T}, jutulV
     sys = ImmiscibleSystem((VaporPhase(), AqueousPhase()), reference_densities = [ρH2O, ρCO2])
     domain = discretized_domain_tpfv_flow(tpfv_geometry(CartesianMesh(M)), porosity = M.ϕ, permeability = M.K)
     model, parameters = setup_reservoir_model(domain, sys, wells = Is)
+    parameters[:Reservoir][:PhaseViscosities][1, :] .= visCO2
+    parameters[:Reservoir][:PhaseViscosities][2, :] .= visH2O
```

Comment:
With the changes in the JutulDarcy.jl dev branch, these two lines can be removed in favor of passing the phase viscosities to `setup_reservoir_model()`, like what is done in https://github.com/sintefmath/JutulDarcy.jl/pull/10 in the test code here: https://github.com/sintefmath/JutulDarcy.jl/pull/10/files#diff-7db9b7a05d8dcf4000c27ce5b02990caffafbabe4cfed529dbdc636f864366edR95

But until those changes get merged into an updated release of JutulDarcy, I wouldn't want to update the syntax here.